dref/models.py

```diff
@@ -375,6 +370,17 @@ class Labels:
         verbose_name=_('organization'),
         help_text=_('Does the NS have Communications capacity?')
     )
+    event_map = models.ForeignKey(
+        'DrefFile', on_delete=models.SET_NULL,
+        null=True, blank=True,
+        verbose_name=_('event map'),
+        related_name=_('event_map')
+    )
+    images = models.ManyToManyField(
+        'DrefFile', blank=True,
+        verbose_name=_('images'),
+        related_name=_('images')
```

Comment:
`related_name` is used for reverse relation. Let's use proper names (and remove translation) here (and above).

https://docs.djangoproject.com/en/3.2/ref/models/fields/#django.db.models.ForeignKey.related_name